config/deploy.rb

```diff
@@ -1,12 +1,12 @@
 # config/deploy.rb file
 require 'bundler/capistrano'
 
-set :application, "scholarship-application-app"
+set :application, "longshot"
 set :deploy_to, ENV["DEPLOY_PATH"]
 server  ENV["SERVER_NAME"], :app, :web
 
-set :user, "ubuntu"
-set :group, "ubuntu"
+set :user, "dosomething"
```

Comment:
Is there a reason we don't just use `ubuntu`/`ubuntu`? So we're not tied to Ubuntu?